src/layout/DefaultLayout.tsx

```diff
@@ -1,15 +1,14 @@
 import { css } from "@emotion/react";
 import { Fragment, PropsWithChildren } from "react";
+import AppBar from "@/component/AppBar/AppBar";
+import { AppBarProps } from "@/component/AppBar/AppBar";
 
-export function DefaultLayout({ children }: PropsWithChildren) {
+type DefaultLayoutProps = PropsWithChildren<AppBarProps>;
+
+export function DefaultLayout({ children, title, appBarVisible, LeftComp, RightComp }: DefaultLayoutProps) {
   return (
     <Fragment>
-      {/* FIXME: 헤더 컴포넌트 작업 시, 해당 헤더 엘리먼트 제거 */}
-      <header
-        css={css`
-          height: 4.6rem;
-        `}
-      />
+      <AppBar title={title} appBarVisible={appBarVisible} LeftComp={LeftComp} RightComp={RightComp} />
```

Comment:
`DefaultLayout` 에서 `appBarVisible` props를 받은 후에, `DefaultLayout`에서 아래와 같이 처리를 할 수도 있을 것 같아요! 이렇게 하면 `<AppBar/>` 컴포넌트에서는 해당 컴포넌트에 필요한 `title`, `LeftComp`, `RightComp`만 넘길 수 있을 것 같아요! 한번 더 props를 넘기는 구조로 설계하신 의도가 따로 있으실까요?! 🤔
```suggestion
      {appBarVisible && <AppBar title={title} LeftComp={LeftComp} RightComp={RightComp} />}
```

Reply:
처음에는 AppBar를 설계할때 Props를 저렇게 설계해놔서 다음과 같이 개발했고, DefaultLayout에 이를 넣으면서 `appBarVisible`을 AppBar에 Props로 넘겨주는게 좋을지, 현우님이 말씀하신 것처럼 하는게 좋을지 고민했었습니다!! 

이렇게 한 이유는 나중에 AppBar만 단독으로 쓰일때 appBarVisible를 쓰기 위한 확장성 때문이었습니다. 
하지만 듣고보니 현우님 말씀대로... 진행하는게 좋겠네요!!